README.md

```diff
@@ -22,8 +22,27 @@ modules:
       # will be auto accepted. Otherwise, all room invites are accepted.
       # Defaults to false.
       accept_invites_only_for_direct_messages: false
+
+      # (For workerised Synapse deployments)
+      # If you want to accept invites on a specific worker, specify its instance
+      # name here. Otherwise, invites will be processed on the main process.
```

Comment:
Is there an ideal process to run this on? (I.e. to avoid replication?) Can we figure this out without asking people to configure more bits? Workers are already quite confusing!

Reply:
The only benefit really would be one of the account data stream writers since it will save a replication trip for DMs (in conjunction with #6), but we're being so pedantic at that point that I thought it would make it more complicated to document that. However we can't autoconfigure that since we don't know the name of an active account data worker.
There's always replication (event creators, event persisters) in the mix somewhere.

Relying on the 'main' process has the advantage of simplicity in that we know there will be exactly one of them and we know it doesn't have a name; it's also fairly understandable that things will break if it's offline.
As you say, workers are confusing, so making it not a problem to configure in the general case seems ideal, hence the default here.